capi.config.ts

```diff
@@ -58,5 +58,21 @@ export const config: CapiConfig = {
         },
       },
     },
+    rococoDevXcm: {
```

Comment:
Can this network be merged with the above once #888 is merged?

Reply:
It can't be merged because `trappist` hasn't been updated to XCM v3.
Therefore this example needs a network with `polkadot v0.9.37`, `polkadot v0.9.370` and `trappist#...`